wp-content/plugins/hurumap/src/FlourishBlock/Edit.js

```diff
@@ -55,7 +146,14 @@ function Edit({
               label: country.name
             }))}
             onChange={country => {
-              setAttributes({ country });
+              setAttributes({
+                country,
+                analysisCountry: country,
+                dataGeoId: `country-${
```

Comment:
Lets change `value` to be `iso_code` instead of slug... Then:
 `dataGeoId: country && (country-${config.countries[country].iso_code}-${config.countries[country].slug})` e.g. `country-KE-kenya`